Manager/src/main/java/io/raspberrywallet/manager/Manager.java

```diff
@@ -69,17 +71,21 @@
         this.tempMonitor = tempMonitor;
         this.wpaConfiguration = new WPAConfiguration();
         this.configuration = configuration;
-
-        Runnable onLockTriggered = () -> {
+        onLockTriggered = () -> {
             frontendChannel.info("Autolock triggered");
             try {
                 lockWallet();
             } catch (WalletNotInitialized ignored) {
                 //we don't care about locking if it wasn't even inited
+            } catch (IncorrectPasswordException | IOException e) {
+                throw new RuntimeException(e);
```

Comment:
should it throw runtime exception? If this contructor is called on unlocking Wallet after power on, then it shouldn't be runtime exception